proposals/0250-NextRpcIndication.md

```diff
@@ -96,6 +96,88 @@ Add the struct to the `SoftButton` struct. This will cover Show/SetDisplayLayout
 </struct>
  ```
 
+### iOS
+1. We're adding in a public manager-level enum
+(iOS example in Swift that we should translate to ObjC)
+```ObjectiveC
+typedef NS_ENUM(NSInteger, NextFunction) {
+        Default =0, 
+        PerformChoiceSet, 
+        Alert, 
+        ScreenUpdate, 
+        Speak, 
+        AccessMicrophone, 
+        ScrollableMessage, 
+        Slider, 
+        SendLocation, 
+        DialNumber, 
+        OpenMenu 
+};
+```
+
+2. Add a manager-level class `NextFunctionInfo` with an initializer 
```

Comment:
This departs from the design we talked about by adding a second class that mirrors and mimics the existing RPC class almost exactly. This kind of code-duplication is unnecessary. You can just use the NextFunctionInfo RPC class and add an initializer to it.

Reply:
Hmm I dont really understand how to modify what I have into what you're talking about here

Reply:
@joeljfischer plz help

Reply:
The design we talked about was simply to add an initializer to the RPC class that takes an `SDLNextFunction` and converts it into the function ID. Your design here is copying the RPC class into a "high-level manager class" with minimal changes. That seems to me to be code duplication without a good reason.